### PR TITLE
[zh] Resync pod-topology-spread-constraints page

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 1]
```diff
@@ -1,64 +1,46 @@
...
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
-
 <!--
 You can use _topology spread constraints_ to control how {{< glossary_tooltip text="Pods" term_id="Pod" >}} are spread across your cluster among failure-domains such as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
 -->
-
-可以使用*拓扑扩展约束*来控制 {{< glossary_tooltip text="Pods" term_id="Pod" >}} 在集群内故障域（例如地区，区域，节点和其他用户自定义拓扑域）之间的分布。这可以帮助实现高可用以及提升资源利用率。
+你可以使用拓扑分布约束（Topology Spread Constraints）来控制
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:18]: I can do it, but italic in Chinese looks strange.

File: content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 2]
```diff
@@ -1,64 +1,46 @@
...
-### 节点标签
+## 先决条件   {#prerequisites}
+
+### 节点标签   {#node-labels}
 
 <!--
 Topology spread constraints rely on node labels to identify the topology domain(s) that each Node is in. For example, a Node might have labels: `node=node1,zone=us-east-1a,region=us-east-1`
 -->
-拓扑扩展约束依赖于节点标签来标识每个节点所在的拓扑域。例如，一个节点可能具有标签：`node=node1,zone=us-east-1a,region=us-east-1`
+拓扑扩展约束依赖于节点标签来标识每个节点所在的拓扑域。
```
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:37]: Done.
👍

**Review by tanjunchen — COMMENTED** [review]
/approve
/lgtm

**Comment by tengqm** [comment]
recommit to trigger CI gate.

**Pull request merged**